pyproject.toml

```diff
@@ -8,8 +8,15 @@ authors = [
 dependencies = [
     "phiflow>=2.4.0",
     "tensorflow>=2.13.0",
-    "tensorflowjs>=4.10.0",
+    "tensorflowjs",
```

Comment:
Needs further digging on the tfjs converter, I am aware of the capability with browser